src/index.ts

```diff
@@ -113,6 +113,13 @@ restInterface.addStatusQuery("external-io-config", () => {
     });
 });
 
+restInterface.addStatusQuery("battery", () => {
```

Comment:
Should we call this battery-status to keep with the naming convention?

Reply:
Yeah I can do that